src/main.cpp

```diff
@@ -188,6 +190,13 @@ void loop() {
         fitnessAppSwitcher.paginationEnable();
         mainAppSwitcher.registerApp(&fitnessAppSwitcher);
         // tools
+#if TOOL_FLASHLIGHT == 1
+        mainAppSwitcher.registerApp(new OswAppFlashLight());
+#endif
+
+#if TOOL_CALCULATOR == 1
+        mainAppSwitcher.registerApp(new OswAppCalculator());
+#endif
```

Comment:
You should move that into the same switcher like the the watch faces - but for tools. Combined with the water-level tool, so we have a new "tool"-section, without slowing the app-cycling too much down. :)

Reply:
sounds like a good idea, but unfortunately it doesent work. Because I need a short press of button 1 in the apps, so it cannot be used to switch between them. I have no better idea, the 3 buttons limit me ther. But I agree that having that many apps in the current app switcher is problematic. Maybe we can come up with a soultion that involves pressing two buttons simultaneously.

Reply:
Okay. This is an argument. We'll add them for now, but I'll also create an issue to get the UI under control :P